app/src/androidTest/java/com/tomtom/sdk/examples/maps/MapLoadingTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.tomtom.sdk.examples.maps
+
+import android.os.Build
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObjectNotFoundException
+import androidx.test.uiautomator.Until
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class MapLoadingTest {
+
+    @get: Rule
+    val activityRule : ActivityScenarioRule<ConfigurableMapActivity> = ActivityScenarioRule(ConfigurableMapActivity::class.java)
+
+
+    @Test
+    fun testMapShowing() {
+        val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
```

Comment:
Duplicated at the beginning of each test

Reply:
Put in @Before set up function.